requirements.txt

```diff
@@ -2,27 +2,27 @@ https://github.com/wazo-platform/wazo-lib-rest-client/archive/master.zip
 https://github.com/wazo-platform/wazo-auth-client/archive/master.zip
 https://github.com/wazo-platform/xivo-lib-python/archive/master.zip
 https://github.com/wazo-platform/xivo-bus/archive/master.zip
-alembic==1.0.0
-celery==4.4.6
-cheroot==6.5.4
-flask-cors==3.0.7
-flask-restful==0.3.7
-flask==1.0.2
-httpx==0.7.2
-importlib_metadata==1.6.0  # from kombu
-itsdangerous==0.24  # from flask
-jinja2==2.10
-jsonpatch==1.21
-kombu==4.6.11  # from xivo-bus
-markupsafe==1.1.0  # from flask
+alembic==1.4.3
+celery==5.0.0
+cheroot==8.5.2
+flask-cors==3.0.9
+flask-restful==0.3.8
+flask==1.1.2
+httpx==0.16.1
+h2==4.0.0  # from httpx
```

Comment:
Same comment that above, if you really need to pin the version can you add why/justification in the commit

Reply:
h2 is required now for httpx to use http2.